content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -90,9 +93,9 @@ Create the invoice
 ==================
 
 When the Helpdesk ticket is completed and the client is ready to be billed for time, begin by
-clicking the :guilabel:`Sales Order` smart button on the ticket form to navigate to the :abbr:`SO
-(Sales Order)`. The :guilabel:`Delivered` column should match the number of hours recorded on the
-ticket's timesheet. After checking and filling out the relevant information, click
+clicking the :guilabel:`Sales Order` smart button on the ticket form to navigate to the
+:guilabel:`Sales Order`. The :guilabel:`Delivered` column should match the number of hours recorded
```

Comment:
I'm not sure if this should be a capitalized guilabel or not. From what I read in this paragraph, it does not seem to be part of the interface, only the name of the object the user will be redirected to.
```suggestion
sales order. The :guilabel:`Delivered` column should match the number of hours recorded
```

Reply:
That makes sense! I'll go ahead and remove the guilabel on that and tag you again once I do.